tests/ui/conftest.py

```diff
@@ -95,29 +297,29 @@ def user(create_superuser, my_base_url, fxa_account, jwt_token):
 @pytest.fixture(scope='function')
 def live_server(request, transactional_db):
```

Comment:
The fixture ```addon``` creates an addon with a specific author. I can only create it once per ```live_server``` session without generating a duplicate key error. 

Now if I understand right, you're saying the fixture ```addon``` should create whatever it needs and return that to the test to be used, say when the tests checks the author.